src/components/Carrousel/Carrousel.tsx

```diff
@@ -31,44 +59,27 @@ const useStyles = makeStyles(theme => ({
 
 const Carrousel: React.FC<{ newbies: Newbie[] }> = ({ newbies }) => {
   const classes = useStyles();
-  const breakpointSm = useMediaQuery<Theme>(theme => theme.breakpoints.down('xs'));
 
   return (
-    <Grid
-      container
-      spacing={2}
-      wrap={breakpointSm ? 'nowrap' : 'wrap'}
-      className={classes.carrousel}>
-      {newbies.map(newbie => {
-        const startDate = new Date(newbie.startDate);
-        return (
-          <Grid key={newbie.id} item>
-            <Link
-              className={classes.link}
-              to={ROUTES.BUDDY_TASKS_LIST.replace(':newbieId', newbie.id)}>
-              <Paper className={classes.carrouselElement}>
+    <Grid container wrap={'wrap'} justify='center' className={classes.carrousel}>
+      {newbies.map(newbie => (
+        <Grid key={newbie.id} item className={classes.carrouselItem}>
+          <Link
+            className={classes.link}
+            to={ROUTES.BUDDY_TASKS_LIST.replace(':newbieId', newbie.id)}>
+            <Paper className={classes.carrouselElement}>
```

Comment:
In MaterialUI we have a `Card` component, It seems the perfect candidate if we take this way by removing the carrousel.